meetings/2025-10-14 - 2025-10-15/2025-10-14-minutes.md

```diff
@@ -0,0 +1,208 @@
+    * A2 (from another contributor): what is working currently is that their organization pays for it  
+    * A3 (another contributor): ISO doesn’t provide its committees with any funding, but it helps them with infrastructure  
+    * A4 (idem): “we currently latch on to Rust conferences to do our meetups (see: All-Hands in Utrecht in May 2025)”. This helps because it spreads the fixed costs of travel between multiple activities. On the other hand, the Rust conferences don’t have a consistent / established cadence or schedule, so it’s a bit inconsistent for now in that format.  
+* Swag Expenses  
+  * How to make them sustainable?  
+    * A1: we’ve shared the designs (in our other committee), in a way such that you can upload designs to a sticker printer, and if anyone wants to get some, they can  
+    * A2: reimbursing the costs with your company is usually a waste of labor \- it’s best to just fund swag in some other way  
+    * A3: manufacturing stickers and such is almost pure fixed cost \- making them en masse is way cheaper per sticker.  
+    * Bonus A: “making the assets available is an instant win without any drawbacks” 🙂  
+* Liaison Subcommittee  
+  * Expert Sessions  
+    * 40-50 attendants for Jill Britton’s talk  
+      * Folks seem to know the basics of the pipeline now, that’s great\!  
+      * [Final minutes here](https://github.com/rustfoundation/safety-critical-rust-consortium/blob/main/subcommittee/liaison/meetings/2025-10-02/minutes.md)  
+    * Requests  
+      * For a talk on the role of a lang standard or spec in certified development  
+      * Tentative:  
+        * Software “quality” metrics?  
+        * Integration of adopted code?  
+      * **If you have questions**:  
+        * DM Alex Celeste on Zulip or email or Github.  
+        * Raising questions in a public space is better \- that’s preferred since different folks might have the same questions  
+  * ISO WG23  
+    * The WG has agreed to start work on a Rust base document  
+      * The WG doesn’t feel like they have any fluent enough Rust experts.  
+      * **Call is out for experts from our group\!**  
+        * Alex currently has 1-2 volunteers from the Rust SC committee.  
+        * Getting to about 4-5 would be a good amount.  
+          The more the merrier, though\!  
+        * **We may want a new SC subcommittee**  
+          * Concerns were raised w.r.t. making such a subcommittee: it was done (sort of) for C and the C folks mostly quit it. Yet the WG found someone who wanted to work on it and it’s... currently a bit awkward.  
+          * Or perhaps a sub sub committee owned by the Guidelines subcommittee  
+    * Do not fret: the WG does not require an ISO standard to exist for the languages they work in (their Python work stands as precedent)  
+  * MISRA  
+    * Still considering a publication in this space  
+  * WG21 / C++  
+    * Continues to collaborate w.r.t. ABI  
```

Comment:
```suggestion
    * Continues to collaborate w.r.t. C++’s ABI
      * Interop with this ABI is not trivial
      * The work is centered around the [Rust-C++ Interoperability Initiative](https://rustfoundation.org/interop-initiative)
```

I've added some extra details so that the reader can get the full context if they are not familiar with this work